en/docs/install-and-setup/setup/distributed-deployment/synchronizing-artifacts-in-a-gateway-cluster.md

```diff
@@ -147,3 +153,33 @@ password = "wso2carbon"
 
 Add the tables of the `AM_GW_API_ARTIFACTS` and `AM_GW_PUBLISHED_API_DETAILS` to this new database that you are specifying. 
 The scripts to create these tables are in the `<API-M_HOME>/dbscripts/apimgt/` directory.
+
+
+## FAQ About Inbuilt Artifact Synchronizer
+
+### In this architecture, does the Gateway requires a database connection?
+No. Gateway does not need to connect with the database to pull the artifacts. It requests from the Traffic Manager and Traffic Manager connects with the database. In this architecture
+Traffic Manager plays a vital roles and maintain the connections with the database.
+
+### If 1st request (i.e saving artifacts to DB) failed and 2nd request (i.e events to TM) success, what will happen? Is this working as a transactional base? (optimistic or pessimistic)
+This will not happen as the event to the Traffic Manager with the published Gateway label will only be sent after a successful DB update. 2nd request will not be sent if the 1st step fails.
```

Comment:
```suggestion
This will not happen as the event to the Traffic Manager with the published Gateway label will only be sent after a successful DB update. The second request will not be sent if the first step fails.
```

Reply:
Fixed with 1269526